Yoast/javascript
# Add an edit icon button

## Summary
Might be an idea to change the edit icon with the new edit icon since this was proposed in Slack. There is a possibility to do this in a later PR though.

The edit icon button can be created by using the variant 'edit'.


This PR can be summarized in the following changelog entry:

* [@yoast/components] Add a new button with an edit icon.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:
* Open the example components app
* Visit the "Reactified components" page
* Check the new edit button
* Think that it looks swell :dancers: 

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance
